src/pages/jinminYang/Login/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Login.scss";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+
+const Login = () => {
+  const [id, setId] = useState(null);
+  const [pw, setPw] = useState(null);
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  return (
+    <div className="login">
+      <div className="compNameDiv">
+        <span className="compName">westagram</span>
+      </div>
+      <form>
+        <input
+          className="id"
+          type="text"
+          placeholder="전화번호, 사용자 이름 또는 이메일"
+          onChange={e => saveUserId(e)}
```

Comment:
```suggestion
          onChange={saveUserId}
```
이럴 때는 이벤트는 이벤트가 발생했을 때 default기 때문에 넣어주지 않아도 동작 됩니다
그러나 이벤트 외에도 다른 인자를 같이 줘야 할 때는 구현하신 방법으로 해주셔야 합니다!